jobsub/examples/pALPIDEfs/steering-templates_pALPIDEfs_7/clusterAnalysis-tmp.xml

```diff
@@ -0,0 +1,146 @@
+  <!--This is the name of the file which contains the pixels which were masked during datataking"-->
+  <parameter name="NoiseMaskFileName" type="string" value="@DatabasePath@/@FilePrefix@-maskedPixels_@dutID@.txt" />
+  <!-- ID of the DUT -->
+  <parameter name="dutID" type="int" value="@dutID@"/>
+  <!--Specify the maximum amount of sectors-->
+  <parameter name="nSectors" type="int" value="8"/>
+  <!--The total amount of pixels the pixels of a cluster have to be away from a sectorborder, for the cluster to be ignored. Also, the distance to the boundary of a chip, for a cluster not to be ignored.-->
+  <parameter name="SectorSafetyPixels" type="int" value="2"/>
+  <!--The total amount of pixels a hotpixel has to be close to a cluster, for the cluster to be ignored.-->
+  <parameter name="LimitAssoc" type="int" value="2"/>
+  <!--Particle energy-->
+  <parameter name="Energy" type="double" value="@BeamEnergy@"/>
+  <!--Chip IDs-->
+  <parameter name="ChipID" type="stringVec" value="@ChipIDs@"/>
+  <!--Irradiation level-->
+  <parameter name="Irradiation" type="stringVec" value="@IrradiationLevel@"/>
+  <!--Data taking rate-->
+  <parameter name="Rate" type="string" value="@Rate@"/>
+  <!--parameter name="Verbosity" type="string" value="DEBUG"/-->
+</processor>
+
+
+ <processor name="Save" type="EUTelOutputProcessor">
+ <!--Writes the current event to the specified LCIO outputfile. Eventually it adds a EORE at the of the file if it was missing Needs to be the last ActiveProcessor.-->
+  <!--drops the named collections from the event-->
+  <parameter name="DropCollectionNames" type="StringVec"> rawdata status noise pedestal data zsdata </parameter>
+  <!--drops all collections of the given type from the event-->
+  <!--parameter name="DropCollectionTypes" type="StringVec"> SimTrackerHit SimCalorimeterHit </parameter-->
+  <!-- write complete objects in subset collections to the file (i.e. ignore subset flag)-->
+  <!--parameter name="FullSubsetCollections" type="StringVec" value="MCParticlesSkimmed"/-->
+  <!--force keep of the named collections - overrules DropCollectionTypes (and DropCollectionNames)-->
+  <!--parameter name="KeepCollectionNames" type="StringVec" value="MyPreciousSimTrackerHits"/-->
+  <!-- name of output file -->
+  <parameter name="LCIOOutputFile" type="string" value="@LcioPath@/run@RunNumber@-noise.slcio"/>
+  <!--write mode for output file:  WRITE_APPEND or WRITE_NEW-->
+  <parameter name="LCIOWriteMode" type="string" value="WRITE_NEW"/>
+  <!--Set it to true to remove intermediate EORE in merged runs-->
+  <parameter name="SkipIntermediateEORE" type="bool" value="true"/>
+  <!--will split output file if size in kB exceeds given value - doesn't work with APPEND and NEW-->
+  <!--parameter name="SplitFileSizekB" type="int" value="1992294"/-->
+  <!--verbosity level of this processor ("DEBUG0-4,MESSAGE0-4,WARNING0-4,ERROR0-4,SILENT")-->
+  <!--parameter name="Verbosity" type="string" value=""/-->
+</processor>
```

Comment:
You can remove completely the save processor since this is a final step of the analysis.